UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.Common.UnitTests/Helpers/NMHelperTest.cs

```diff
@@ -0,0 +1,425 @@
+                        Assert.AreEqual(expected[i].DailyFilesData[j].FileSizeinKB, result[i].DailyFilesData[j].FileSizeinKB);
+                        Assert.AreEqual(expected[i].DailyFilesData[j].MimeType, result[i].DailyFilesData[j].MimeType);
+                        Assert.AreEqual(expected[i].DailyFilesData[j].Links, result[i].DailyFilesData[j].Links);
+                        Assert.AreEqual(expected[i].DailyFilesData[j].AllFilesZipSize, result[i].DailyFilesData[j].AllFilesZipSize);
+                    }
+                }
+            });
+        }
+
+        [Test]
+        public void WhenGetFormattedDateIsCalled_ThenShouldReturnDateInExpectedFormat()
+        {
+            string strdate = "2022-05-26";
+            string actualDate = NMHelper.GetFormattedDate(strdate);
+            string expectedDate = "26-05-22";
+            Assert.AreEqual(expectedDate, actualDate);
+        }
+
+        private static BatchSearchResponse SetSearchResultForWeekly()
+        {
+            BatchSearchResponse SearchResult = new()
+            {
+                Count = 2,
+                Links = null,
+                Total = 0,
+                Entries = new List<BatchDetails>() {
+                    new BatchDetails() {
+                        BatchId = "1",
+                        Files = new List<BatchDetailsFiles>() {
+                            new BatchDetailsFiles () {
+                                Filename = "aaa.pdf",
+                                FileSize=1232,
+                                MimeType = "PDF",
+                                Links = null
+                            },
+                            new BatchDetailsFiles () {
+                                Filename = "bbb.pdf",
+                                FileSize=1232,
+                                MimeType = "PDF",
+                                Links = null
+                            }
+                        }
+
+                    },
+                    new BatchDetails() {
+                        BatchId = "2",
+                        Files = new List<BatchDetailsFiles>() {
+                            new BatchDetailsFiles () {
+                                Filename = "ccc.pdf",
+                                FileSize=1232,
+                                MimeType = "PDF",
+                                Links = null
+                            },
+                            new BatchDetailsFiles () {
+                                Filename = "ddd.pdf",
+                                FileSize=1232,
+                                MimeType = "PDF",
+                                Links = null
+                            }
+                        }
+
+                    }
+                }
+            };
+
+            return SearchResult;
+        }
+
+        private static BatchSearchResponse SetSearchResultForDaily()
+        {
+            BatchSearchResponse SearchResult = new()
```

Comment:
Camel case variable names please:
```suggestion
            BatchSearchResponse searchResult = new()
```